protected/models/SurveyLanguageSetting.php

```diff
@@ -92,6 +92,8 @@ public function relations()
     public function rules()
     {
         return [
+            ['surveyls_survey_id','required', 'on' => 'insert'],
+            ['surveyls_language','required', 'on' => 'insert'],
```

Comment:
Rules are not checked at creation. Just set survey Id after construction
and it should work.

On Fri, Oct 9, 2015, 11:56 Denis Chenu <notifications@github.com> wrote:

> In protected/models/SurveyLanguageSetting.php
> <https://github.com/LimeSurvey/LimeSurvey/pull/373#discussion_r41614679>:
>
> > @@ -92,6 +92,8 @@ public function relations()
> >      public function rules()
> >      {
> >          return [
> > +            ['surveyls_survey_id','required', 'on' => 'insert'],
> > +            ['surveyls_language','required', 'on' => 'insert'],
>
> ... must be fixed : no ability to create SUrvey : $this->survey is null at
> create
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/LimeSurvey/LimeSurvey/pull/373/files#r41614679>.
>